libs/hw---stm32f401/pxt.json

```diff
@@ -32,7 +32,9 @@
     "yotta": {
         "config": {
             "DEVICE_USB": 1,
-            "PXT_DEFUALT_ACCELEROMETER": "ACCELEROMETER_TYPE_MMA8453"
+            "DEVICE_WEBUSB": 1,
+            "DEVICE_JACDAC_DEBUG": 1,
+            "PXT_DEFAULT_ACCELEROMETER": "ACCELEROMETER_TYPE_MMA8453"
```

Comment:
@pelikhan I changed this line as there was a typo.

Reply:
Can revert if the previous config was correct...

Reply:
it was a typo